### ISSUE TITLE
Suggest rewording for search optimization

### ISSUE THREAD

**Issue by ak77-prime** [magento/knowledge-base#166]
# Issue: Topic clarification/correction
 
## Topic Link
 
https://support.magento.com/hc/en-us/articles/6706549261709
 
## Description
 
I sometimes have difficulty locating this article because I keep searching for "wrong account"

It would help to also include "wrong" in the article as a keyword to aid in the search; perhaps reword the Issue description, e.g.,

```
When you log in to your cloud account at https://accounts.magento.cloud/user, you are automatically switched to the wrong account associated with another user/email address.

As a result, you are unable to access the correct project associated with your account.
```

**Comment by barnato-v** [comment]
This issue was fixed in PR here https://github.com/magento-commerce/knowledge-base/pull/1402
